strawberry/types/type_resolver.py

```diff
@@ -280,22 +280,36 @@ def _get_fields(cls: Type) -> List[FieldDefinition]:
     """
     field_definitions: Dict[str, FieldDefinition] = {}
 
-    # Type #1 fields
+    # before trying to find any fields, let's first add the fields defined in
+    # parent classes, we do this by checking if parents have a type definition
+    for base in cls.__bases__:
+        if hasattr(base, "_type_definition"):
+            base_field_definitions = {
+                field.name: field
+                # TODO: we need to rename _fields to something else
+                for field in base._type_definition._fields  # type: ignore
+            }
+
+            # Add base's field definitions to cls' field definitions
+            field_definitions = {**field_definitions, **base_field_definitions}
+
+    # then we can proceed with finding the fields for the current class
```

Comment:
I'm surprised this worked before, this should have been always done before find all the other fields